docs/troubleshooting.md

```diff
@@ -9,3 +9,16 @@ To troubleshoot Kangal you will need access to your Kubernetes cluster.
 
 ## Problems with a specific load test
 You can make basic troubleshooting using Kangal API endpoints or either exploring load test Pods if is the case of your backend.
+
+- Get status of the load test 
+```bash
+curl -X GET 'http://${KANGAL_PROXY_ADDRESS}/load-test/loadtest-random-name/' 
+```
+- Get logs from the master pod
+```bash
+curl -X GET 'http://${KANGAL_PROXY_ADDRESS}/load-test/loadtest-random-name/logs' 
+```
+- Get logs from the worker pod
+```bash
+curl -X GET 'http://${KANGAL_PROXY_ADDRESS}/load-test/loadtest-random-name/logs/loadtest-worker-000' 
```

Comment:
```suggestion
curl -X GET 'http://${KANGAL_PROXY_ADDRESS}/load-test/loadtest-random-name/logs/worker-000' 
```
I think it will be better if we not repeat the `loadtest-` prefix again here, it's too much repetition.